internal/expr/prepare.go

```diff
@@ -50,6 +51,9 @@ func getKeys[T any](m map[string]T) []string {
 	return keys
 }
 
+// starCount counts the number of fullNames in the list with asterisks in the
+// name field. It is used to check if there are any asterisk expressions in a
+// list of types or columns.
```

Comment:
```
// starCount counts the number of fullName values that have "*" as a name.
```
Note:
- fullName is a type name, so we can't make it plural.
- The "in" was misleading: it must _be_ an astarisk rather than _contain_ one.
- It's an anti-pattern to document who calls a function and why. We document what it does instead, and whoever needs it can call it..

Reply:
Good points, changed.